multi-int-store/src/main/java/com/connexta/multiintstore/common/ProductStorageManager.java

```diff
@@ -51,6 +51,7 @@ public URI storeProduct(
    * RetrieveResponse}.
    */
   public RetrieveResponse retrieveProduct(@NotEmpty String id) throws StorageException {
-    return storageAdaptor.retrieve(id);
+    // return storageAdaptor.retrieve(id);
+    return null;
```

Comment:
:question: Should this be returning null?

Reply:
No, no it should not. Thanks!